src/main/resources/org/sagebionetworks/web/client/widget/entity/tabs/FilesTabViewImpl.ui.xml

```diff
@@ -41,6 +41,14 @@
 			    		<bh:Div ui:field="fileProvenanceContainerHighlightBox" />
 			    	</g:FlowPanel>
 				</b:Column>
+				<b:Column size="MD_12" ui:field="discussionContainer">
+					<g:FlowPanel addStyleNames="light-border padding-10 margin-bottom-15">
+			    		<bh:Div addStyleNames="highlight-title">
+			    			<bh:Text>Discussion about this entity</bh:Text>
```

Comment:
Let's not expose "entity".  Please either reduce to simply "Discussion", or "Discussion about this <Entity type friendly name>".